test-suites/gutenberg/sanity-test-suites.md

```diff
@@ -98,17 +98,17 @@ This holds a grouping of certain test suites to run in order to share the work w
 
 ### Group - 1
 
-- [ ] Group - Deep nesting is possible (iOS only) - [TC001](https://github.com/wordpress-mobile/test-cases/blob/master/test-cases/gutenberg/group.md#tc001)
+- [ ] Group - Deep nesting is possible - [TC001](https://github.com/wordpress-mobile/test-cases/blob/master/test-cases/gutenberg/group.md#tc001)
```

Comment:
This was iOS only due to a limitation in the React Native implementation for iOS where having too many nested `Views` would generate a crash. Is this also now happening on Android for the Group block?

Reply:
@geriux are you saying this test was verifying that a crash did not occur on iOS? If so, I misinterpreted the "iOS only" note to mean "deploying nesting is _only possible_ on iOS." So, there is room to improve how we phrase this note. 

We could either (A) remove the note to keep things simple or (B) expand the phrasing to be more clear, e.g. "test iOS specifically."

Reply:
I updated the existing note to clarify the intent that iOS needs to be tested specifically for a crash.